src/lib/workspace/Variable.tsx

```diff
@@ -31,32 +32,88 @@ export function VariableDetails(props: Props) {
+        groupBy(entity.variables, (variable) => variable.parentId)
+      ).map((variable, i) => {
+        if (i < 3) {
+          return (
+            <div key={variable.id}>
+              {variable.displayName}:{' '}
+              {variable.providerLabel
+                .replace(/[\[\]"]/g, '')
+                .replace(/[,]/g, ', ')}
+              ;&nbsp;
+            </div>
+          );
+        }
+      })
+    : variable.providerLabel.replace(/[\[\]"]/g, '').replace(/[,]/g, ', ');
+
+  // make variables for after the first three
+  const providerLabelLeftover = MultiFilterVariable.is(variable)
+    ? findMultifilterVariableLeaves(
+        variable,
+        groupBy(entity.variables, (variable) => variable.parentId)
+      ).map((variable, i) => {
+        if (i > 2) {
+          return (
+            <div key={variable.id}>
+              {variable.displayName}:{' '}
+              {variable.providerLabel
+                .replace(/[\[\]"]/g, '')
+                .replace(/[,]/g, ', ')}
+              ;&nbsp;
+            </div>
+          );
+        }
+      })
+    : '';
+
+  // define show more link
+  const showMoreLink = showMore ? 'Show Less << ' : 'Show More >> ';
+
   return (
     <ErrorBoundary>
       <div>
         <h3>{axisLabelWithUnit(variable)}</h3>
         <div className={cx('-ProviderLabel')}>
           <div className={cx('-ProviderLabelPrefix')}>
-            Original variable{' '}
-            {MultiFilterVariable.is(variable) ? 'names' : 'name'}:
+            <i>
+              Original variable{' '}
+              {MultiFilterVariable.is(variable) ? 'names' : 'name'}:
+            </i>
           </div>
-          &nbsp;
-          {MultiFilterVariable.is(variable)
-            ? findMultifilterVariableLeaves(
-                variable,
-                groupBy(entity.variables, (variable) => variable.parentId)
-              ).map((variable) => (
-                <div key={variable.id}>
-                  {variable.displayName}: {variable.providerLabel};&nbsp;
-                </div>
-              ))
-            : variable.providerLabel}
-          &nbsp;
+          {/* showing three variables for multifilter or single variable */}
+          &nbsp; {threeProviderLabel} &nbsp;
           <HelpIcon>
-            The name for this variable as provided with the original study's
-            data set. The VEuPathDB team curates variable names and places
-            variables into an ontology framework.
+            The name of this variable in the data files that were integrated
+            into ClinEpiDB
           </HelpIcon>
+          &nbsp;&nbsp;
+          {MultiFilterVariable.is(variable) ? (
+            <>
+              <a
+                className="variable-show-more-link"
+                style={{ cursor: 'pointer' }}
+                onClick={() => {
+                  setShowMore(!showMore);
+                }}
+              >
+                {showMoreLink}
+              </a>
```

Comment:
Instead of rendering an `a`nchor with an inline `cursor: pointer` style, you can use a `button` with a `link` class:

```suggestion
              <button
                className="variable-show-more-link link"
                onClick={() => {
                  setShowMore(!showMore);
                }}
              >
                {showMoreLink}
              </button>
```

("link" is a utility class offered by [@veupathdb/web-common](https://github.com/VEuPathDB/EbrcWebsiteCommon/blob/6a4cd50198a0aaa17c78961716e9975ea6afbef9/Client/src/styles/AllSites.scss#L93) which can be used to style a button as though it were a link.)

Reply:
@jtlong3rd thank you for your review and suggestions 👍 Actually I was also thinking of the use of button but just did it with <a> 😄 Will change it to <button> as suggested